inst/htmlwidgets/plugins/leaflet-draw-plugin/leaflet-draw-plugin.js

```diff
@@ -0,0 +1,19 @@
+  LeafletWidget.methods.addDrawToolbar = function(){
```

Comment:
Do you mind cleaning up the indentation in this file? Two spaces per level, please! :)